client/packages/requisitions/src/ResponseRequisition/DetailView/columns.ts

```diff
@@ -77,8 +66,8 @@ export const useResponseColumns = () => {
       align: ColumnAlign.Right,
       getSortValue: rowData =>
         rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand ?? '',
-      Cell: PackVariantQuantityCell({
-        getItemId: row => row.itemId,
+      Cell: PackSizeQuantityCell({
```

Comment:
Possibly can just remove `PackSizeQuantityCell` or create something more generic like `QuantityCell`?

Reply:
It's not compiling (didn't fix everywhere) but have this in my suggestion branch, what do you reckon? I think worth taking out the indirection...

https://github.com/msupply-foundation/open-msupply/commit/be37b069dd4d59aa24c5f7a9d460dc2d4e511839

Reply:
Yeah I think this is probably worth doing. Do you want me to go through finding more places? Maybe we just merge your branch and be done? Was trying not to do too much work on this as it already added up to a lot of time :(

Reply:
Yeah let's not spend too much time, will need to fix the errors but otherwise sounds good 🙏